system/control-plane/cell/components/cell-sts/src/main/java/org/wso2/vick/auth/cell/jwks/SPIFFEKeyResolver.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.vick.auth.cell.jwks;
+
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+/**
+ * Resolves keys derived in the format of SPIFFE
+ */
+public class SPIFFEKeyResolver extends StaticKeyResolver {
+
+    // Sample SPIFFE public certificate string
+    private static String publicKeyString = "-----BEGIN CERTIFICATE-----\n" +
```

Comment:
Just to know, why have we hardcoded this public key string?

Reply:
This is just a sample SPIFFE certificate.. If we need we can use this for testing purposes.. Later we need to get dynamically distrusted certs..